src/card/card-actions.jsx

```diff
@@ -24,8 +24,10 @@ let CardActions = React.createClass({
       });
     });
 
+    let mergedStyles = this.mergeAndPrefix(styles.root, this.props.style);
```

Comment:
But it's a `let` [in](https://github.com/louy/material-ui/blob/patch-1/src/card/card-header.jsx#L61) [other](https://github.com/louy/material-ui/blob/patch-1/src/card/card-title.jsx#L50) [components](https://github.com/louy/material-ui/blob/patch-1/src/card/card.jsx#L53).